command/crypto/jws/sign.go

```diff
@@ -194,14 +197,36 @@ func signAction(ctx *cli.Context) error {
 		if len(x5cKeyFile) == 0 {
 			return errs.RequiredWithOrFlag(ctx, "x5c-cert", "key", "x5c-key")
 		}
+		if len(x5tCertFile) > 0 {
+			return errs.MutuallyExclusiveFlags(ctx, "x5c-cert", "x5t-cert")
+		}
 		if ctx.IsSet("jwk") {
 			return errs.MutuallyExclusiveFlags(ctx, "x5c-cert", "jwk")
 		}
 		if len(jwks) > 0 {
 			return errs.MutuallyExclusiveFlags(ctx, "x5c-cert", "jwks")
 		}
 		isX5C = true
-	} else {
+	}
+
+	var isX5T bool
+	if len(x5tCertFile) > 0 {
+		if len(x5tKeyFile) == 0 {
+			return errs.RequiredWithOrFlag(ctx, "x5t-cert", "key", "x5t-key")
+		}
+		if len(x5cCertFile) > 0 {
+			return errs.MutuallyExclusiveFlags(ctx, "x5t-cert", "x5c-cert")
+		}
+		if ctx.IsSet("jwk") {
+			return errs.MutuallyExclusiveFlags(ctx, "x5t-cert", "jwk")
+		}
+		if len(jwks) > 0 {
+			return errs.MutuallyExclusiveFlags(ctx, "x5t-cert", "jwks")
+		}
+		isX5T = true
+	}
+
+	if !isX5C || !isX5T {
```

Comment:
Looks like the proper condition here is `&&`. This is not causing a problem right now because `key` won't be empty because is an alias of `x5c-key` and `x5t-key`

Reply:
good spot